.dotnet/api/OpenAI.netstandard2.0.cs

```diff
@@ -49,6 +51,8 @@ public class AssistantClient {
         public AssistantClient(ApiKeyCredential credential, OpenAIClientOptions options);
         public AssistantClient(ApiKeyCredential credential);
         protected internal AssistantClient(ClientPipeline pipeline, OpenAIClientOptions options);
+        public AssistantClient(string apiKey, OpenAIClientOptions options);
+        public AssistantClient(string apiKey);
```

Comment:
BatchClient, FineTuningClient, ModelClient, ModerationClient, VectorStoreClient

Reply:
Unless those are related to the SCM 1.1.0 update, we should defer those changes to a different PR.

Reply:
Sounds good